src/utils/cleanupFilesForSurf.ts

```diff
@@ -0,0 +1,47 @@
+import fs from "node:fs";
+import { Selections } from "../types.js";
+import { TemplateFramework, TemplateProjectType } from "./constants.js";
+import { move, remove } from "./helpers.js";
+
+export const cleanupFilesForSurf = (selection: Selections) => {
+  if (selection.projectType === TemplateProjectType.MOVE) {
+    return;
+  }
+
+  let frontend_dir: string;
+  if (selection.framework === TemplateFramework.VITE) {
+    frontend_dir = "frontend";
+  } else if (selection.framework === TemplateFramework.NEXTJS) {
+    frontend_dir = "src";
+  } else {
+    throw new Error("Unsupported framework");
+  }
+
+  if (selection.template.path === "boilerplate-template") {
```

Comment:
```suggestion
  if (selection.template.path === FullstackBoilerplateTemplateInfo.value.path) {
```

Reply:
Also, I think you can simply check this condition at the top of the function, if it is not this path we can simply return